src/metabase/api/pulse.clj

```diff
@@ -1,6 +1,7 @@
 (ns metabase.api.pulse
   "`/api/pulse` endpoints. These are all authenticated. For unauthenticated `/api/pulse/unsubscribe` endpoints,
   see [[metabase.api.pulse.unsubscribe]]."
+  #_{:clj-kondo/ignore [:deprecated-namespace]}
```

Comment:
Sucks that we can't put this on the namespace itself, we should open a cljfmt ticket and see if dev-ex can pick it up.